lib/matplotlib/path.py

```diff
@@ -545,6 +604,35 @@ def get_extents(self, transform=None):
                 transform = None
         return Bbox(_path.get_path_extents(path, transform))
 
+    def get_exact_extents(self, **kwargs):
```

Comment:
Can you just replace the old `get_extents` implementation (unless this one is significantly slower in the polyline case)?  It seems a bit annoying to have both an approximate `get_extents` and a separate `get_exact_extents`.

Reply:
Done. However, this also obviates the `_path.get_path_extents` code. Should this be simply removed (since it's private and no longer used)?

There does not appear to be a significant slowdown of the test suite on my machine after replacing the approximate compiled routine with my exact Python routine, since this is typically only called once per-path per-savefig.

*Also*, this will likely break many image tests, presumedly since it changes what the bbox is which is being used to compute plot bounds, etc...what's the latest on  #16447? Should I add them all back in by hand?

Reply:
Just kidding. Apparently it didn't break any image tests, I had accidentally compiled against system freetype....😅